app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @user = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+end
```

Comment:
- [ ]  There should be a corresponding view file for each controller, kindly add the files in your view folder